src/configmanager.cpp

```diff
@@ -253,6 +253,7 @@ bool ConfigManager::load()
 	string[URL] = getGlobalString(L, "url", "");
 	string[LOCATION] = getGlobalString(L, "location", "");
 	string[WORLD_TYPE] = getGlobalString(L, "worldType", "pvp");
+	string[SPDLOG_LEVEL] = getGlobalString(L, "spdlogLevel", "info");
```

Comment:
I think the logger should be transparent to the user, it doesn't matter if we use spdlog or something else, so just `logLevel` is better

Also, please read from an environment variable `LOG_LEVEL` like database credentials, it's useful for containerizing the server 😉